src/coreclr/vm/prestub.cpp

```diff
@@ -412,6 +412,17 @@ PCODE MethodDesc::PrepareILBasedCode(PrepareCodeConfig* pConfig)
         DACNotifyCompilationFinished(this, pCode);
     }
 
+#if defined(FEATURE_TIERED_COMPILATION) && !defined(TARGET_X86)
```

Comment:
Can you elaborate on the x86 aspect of this?

Reply:
My PR that moves x86 to the full UnmanagedCallersOnly plan still hasn't been merged yet. That's basically it.